web/js/components/timeline/timeline-data/info-modal.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+const DataPanelInfoModal = () => (
+  <div className="data-panel-info">
+    <header>
+      <p>
+        Temporal coverage of activated time-varying layers are shown in the layer coverage panel as striped blue lines in relation to the current timeline zoom level. The total temporal coverage dates are shown on the right.
```

Comment:
"activated time-varying layers" sounds complicated.  Maybe just "coverage of temporal layers"?